test/proxy_test.js

```diff
@@ -107,4 +108,46 @@ describe('proxy middleware should proxy generic requests', function () {
 			});
 		});
 	});
+
+
+	it('should support basic auth', function (done) {
+		var sExpectedResponse = 'All ok!',
+		sExpectedPath = '/foo/bar',
+		iExpectedStatusCode = 200,
+		oAppToBeProxied = connect(),
+		sActualResponse = '',
+		sActualPath,
+		iActualStatusCode;
+
+		oAppToBeProxied.use(function (oRequest, oResponse) {
+			// x-forwareded headers are expected (see xfwd option in proxy config)
+			assert.equal(oRequest.headers['authorization'], 'Basic bWU6cHdk');
```

Comment:
The test still succeeds as this error is not handled anywhere. But it seems that this is also a problem with the other tests.